liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -1323,8 +1327,12 @@ private void doDevMode() throws MojoExecutionException {
             // If non-container, install features before starting server. Otherwise, user
             // should have "RUN features.sh" in their Dockerfile if they want features to be
             // installed.
-            if (!container) {
+            // Added check here for the new skip install feature parameter. 
+            // Need to also check if this is a new Liberty installation or not. The isNewInstallation flag is set by runLibertyMojoCreate.
+            if (!container && (!skipInstallFeature || isNewInstallation)) {
                 runLibertyMojoInstallFeature(null, null, null);
+            } else {
+                getLog().info("Skipping liberty:install-feature");
```

Comment:
Do we like now getting an info message in the devc case that we're skipping?

If we decide we want to separate it out, I might say that we're "Skipping liberty:install-feature because of skipInstallFeature config".   As opposed to the general 'skip' config.

Reply:
Will change this message as discussed and only display it when skipInstallFeature is set to true.

Reply:
Done